qemu/tests/iperf_test.py

```diff
@@ -86,6 +87,19 @@ def is_iperf_running(name_pattern, session=None):
                                     ignore_status=True, verbose=False)
         return status == 0
 
+    def rss_test():
+        logging.info('Run the command "netkvm-wmi.cmd rss" to collect statistics')
+        if params.get("rss_test", "no") == "yes":
```

Comment:
Move this line above line 159.
```python
if params.get("rss_test", "no") == "yes":
    rss_test()
```

Reply:
According to the comments added by the developer in the bug, it is more reasonable to run the command "netkvm-wmi.cmd rss" to collect information after some network activities.